### DISCUSSION TITLE
Installing HOOMD with ENABLE_LLVM option

### DISCUSSION THREAD

**Discussion by dayakaran** [glotzerlab/hoomd-blue#1799] [General]
Hello,

I am trying to build HOOMD 4.7.0 (CPU only) with the ENABLE_LLVM option turned on.

GCC version 11.4
LLVM version 14.0.6

The compilation proceeded without any issues and I was able to install it. 

However when I tried to import hoomd I see some errors.  I used python3 -v to get more information on why I see the error

```
# /home/jedison2/src/miniconda3/lib/python3.12/site-packages/hoomd/__pycache__/_compile.cpython-312.pyc matches /home/jedison2/src/miniconda3/lib/python3.12/site-packages/hoomd/_compile.py
# code object from '/home/jedison2/src/miniconda3/lib/python3.12/site-packages/hoomd/__pycache__/_compile.cpython-312.pyc'
import 'hoomd._compile' # <_frozen_importlib_external.SourceFileLoader object at 0x1554eab1fa70>
: CommandLine Error: Option 'use-dbg-addr' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
Aborted (core dumped)
```

I am only able to find much information on this error. 

Could you please help me with it ?

Please let me know if you need more information from me.

Thanks
John

**Comment by joaander** [comment]
I haven't seen this error before and have no suggestions. `ENABLE_LLVM=on` is deprecated and will not be further supported. When possible, use new `hpmc.pair` potentials that are compiled in. Soon, we will post a template that you can use to compile arbitrary C++ code for use with HPMC.

More information:
* https://github.com/glotzerlab/hoomd-blue/discussions/1745#discussioncomment-8852889
* #1652
